app/subcommands/group/run

```diff
@@ -12,7 +12,10 @@ run_with_dependencies() {
 	configkey="$2"
 	configpath="$(config_path "$configkey")"
 	[ -f "$configpath" ] || return 0
-	xargs -I{} sh -c "$depcmd {}" <"$configpath"
+	while read -r dep; do
+		# shellcheck disable=SC2086
+		$depcmd $dep </dev/null
+	done <"$configpath"
```

Comment:
This does not work for passing arguments in as read and while will cause it to split on all whitespace, not just new lines.

Reply:
Seems to work ok https://github.com/Nekroze/dab/blob/master/tests/features/group.feature#L72 passes

Reply:
huh, then I have no idea why I switched to xargs haha. Too long ago.

Reply:
lol wasn't even me. Love your work guys!